hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -138,14 +154,11 @@ def queries(query_group,
             else:
                return None
 
-
     orig_filename = query_file
-    query_file = __salt__['cp.cache_file'](query_file)
+
     if query_file is None:
         log.error('Could not find file {0}.'.format(orig_filename))
         return None
```

Comment:
This error logic is no longer necessary here -- can we move it up into your `for` loop? We want to log an error in case any of the files passed into the list cannot be found.